projects/samples/howto/controllers/console/console.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 1996-2020 Cyberbotics Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <webots/robot.h>
+
+#include <webots/utils/ansi_codes.h>
+
+#define TIME_STEP 64
+
+int main(int argc, char **argv) {
+  wb_robot_init();
+
+  ANSI_PRINTF_IN_GREEN("This text will not be seen because we are going to clear the console");
```

Comment:
It's a pity that we don't see this message at all and maybe skip the fact that the console is actually cleared.
```suggestion
  ANSI_PRINTF_IN_GREEN("This text will disappear in about 4 seconds because we are going to clear the console...");
  wb_robot_step(4096);
```